node/node.go

```diff
@@ -271,17 +271,18 @@ func activateShhService(stack *node.Node, config *params.NodeConfig, db *leveldb
 		whisperServiceConfig := &whisper.Config{
 			MaxMessageSize:     whisper.DefaultMaxMessageSize,
 			MinimumAcceptedPOW: 0.001,
-			TimeSource:         time.Now,
```

Comment:
Does it mean that `TimeSource` in config is unused and can be removed?

Reply:
in what config? i removed TimeSource from whisper.Config and now it is set with separate method. it was simpler to integrate with tests this way. as for NTPSyncEnabled parameter from status config - it is still used and doesn't need to be removed

Reply:
> i removed TimeSource from whisper.Config

That's what I meant :)